### ISSUE TITLE
New tree symbols block text labels.

### ISSUE THREAD

**Issue by Klumbumbus** [gravitystorm/openstreetmap-carto#1475]
The tree icons block text labels. They should not, as trees are are low priority feature.

Examples: 

Name of school is blocked by trees (the name was displayed before I added the trees): [here](http://www.openstreetmap.org/way/203515999)

Name of dentist is blocked by trees at z17 [here](http://www.openstreetmap.org/node/2261569278#map=17/50.69930/12.85086)

You can also see some disappearing street names in the first 4 example images [here](https://github.com/gravitystorm/openstreetmap-carto/pull/978#issuecomment-72276083)

**Comment by pnorman** [comment]
duplicate of #1425

**Comment by Klumbumbus** [comment]
Sorry for duplicate. I searched only for open issues...
